source/simulator/core.cc

```diff
@@ -2038,11 +2039,15 @@ namespace aspect
 
               assemble_stokes_system();
               build_stokes_preconditioner();
+
+              if (iteration == 0)
+                initial_stokes_residual = compute_initial_stokes_residual();
+
               const double stokes_residual = solve_stokes();
               current_linearization_point = solution;
 
-              pcout << "      Nonlinear Stokes residual: " << stokes_residual << std::endl;
-              if (stokes_residual < 1e-8)
+              pcout << "      Nonlinear Stokes residual: " << stokes_residual/initial_stokes_residual << std::endl;
```

Comment:
Is this how the other solvers do it, i.e., output the *relative* residual? That makes no sense to me, but if that's what they all do, then I guess it makes sense to do the same here...